shortify/redirect_test.go

```diff
@@ -41,6 +47,23 @@ func (suite *RedirectSuite) TestSaveNewRecord() {
 	assert.NotEmpty(t, redir.Token)
 }
 
+func (suite *RedirectSuite) TestTokenMustBeUnique() {
```

Comment:
maybe `TestCreatingNonUniqueTokenReturnsError` ? Does this make sense -- or do we want people always to be able to create a token and have it return here's the token -- in the context of creation it'd just return you the already created token?...